frameworks/C/h2o/src/main.c

```diff
@@ -177,9 +183,7 @@ static int initialize_global_data(const config_t *config, global_data_t *global_
 	sigset_t signals;
 
 	memset(global_data, 0, sizeof(*global_data));
-	global_data->config = config;
 	global_data->memory_alignment = get_maximum_cache_line_size();
```

Comment:
Just to confirm, are you actually caching data, or is your use of "cache line size" an aspect of memory management within h2o?